website/app/styles/pages/components/code-block.scss

```diff
@@ -15,5 +15,13 @@
     border-radius: 3px;
     aspect-ratio: 1/1;
   }
+
+  .doc-code-block-flex-layout {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+    justify-content: space-between;
+    margin-bottom: 1rem;
+  }
```

Comment:
I would probably just add margin styles to space apart the text in your examples vs. making everything a flex layout. It would be simpler & require less code.

Reply:
Oh! good point. Updated.